test/parallel/test-whatwg-encoding.js

```diff
@@ -0,0 +1,94 @@
+// Flags: --expose-internals
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const { Buffer, TextDecoder, TextEncoder } = require('buffer');
```

Comment:
Why do we use it only in the [lib/.eslintrc.yaml](https://github.com/nodejs/node/blob/master/lib/.eslintrc.yaml#L3) and not in the [test/.eslintrc.yaml](https://github.com/nodejs/node/blob/master/test/.eslintrc.yaml)?